index.vue

```diff
@@ -50,6 +50,11 @@ export default
 			type: Number
 			default: 0
 
+
+		offScreenPad:
+			type: Number
+			default: 0
```

Comment:
What would you think about making this `offscreenHeight` instead?  

My reasoning is that I think this makes it easier to visualize what the prop does if it just _replaces_ the height when the header is offscreen rather being an adjustment to the provided `height`.  If so, it could just default to the height.  Like:

```suggestion
		offscreenHeight:
			type: Number
			default: -> @height
```